src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,62 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exception.RegistrationException;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_AGE = 18;
+    private static final int MIN_NUMBER_SYMBOLS = 6;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        verificationOfUserNull(user);
+        verificationOfSameLogin(user);
```

Comment:
Create
validateLogin(user);
validatePassword(user);
validateAge(user);

Don't need to create methods for each case, one method provide logic to validate one point - login , password, age